java/wavefrontsender.md

```diff
@@ -5,103 +5,147 @@
 * [Overview](#Overview)
 * [Option 1: Sending Data via the Wavefront Proxy](#Option-1-Sending-Data-via-the-Wavefront-Proxy)
 * [Option 2: Sending Data via Direct ingestion](#Option-2-Sending-Data-via-Direct-ingestion)
+* [Send Data to Multiple Wavefront Services](#Send-Data-to-Multiple-Wavefront-Services)
 * [Share a WavefrontSender](#Share-a-WavefrontSender)
 * [Where to Go Next](#where-to-go-next)
 
 ## Overview
 
-You can send metrics, histograms, or trace data from your application to the Wavefront service using a Wavefront proxy or direct ingestions.
+You can send metrics, histograms, or trace data from your application to the Wavefront service using a Wavefront proxy or direct ingestion. 
 
 * Use a [**Wavefront proxy**](https://docs.wavefront.com/proxies.html), which then forwards the data to the Wavefront service. This is the recommended choice for a large-scale deployment that needs resilience to internet outages, control over data queuing and filtering, and more.
-* Use [**direct ingestion**](https://docs.wavefront.com/direct_ingestion.html) to send the data directly to the Wavefront service. This is the simplest way to get up and running quickly. 
+* Use [**direct ingestion**](https://docs.wavefront.com/direct_ingestion.html) to send the data directly to the Wavefront service. This is the simplest way to get up and running quickly.
 
-The `WavefrontSender` interface has two implementations:
+Let's create a `WavefrontClient` to send data to Wavefront either via Wavefront proxy or directly over HTTP.
 
-* Option 1: [Create a `WavefrontProxyClient`](#Option-1-Sending-Data-via-the-Wavefront-Proxy) to send data to a Wavefront proxy.
-* Option 2: [Create a `WavefrontDirectIngestionClient`](#Option-2-Sending-Data-via-Direct-ingestion) to send data directly to a Wavefront service.
+> **Deprecated implementations**: *`WavefrontDirectIngestionClient` and `WavefrontProxyClient` are deprecated from proxy version 7.0 onwards. We recommend all new applications to use `WavefrontClient`.*
+
+* Use `WavefrontClientFactory` to create a `WavefrontClient` instance, which can send data directly to a Wavefront service or send data using a Wavefront Proxy.
+* The `WavefrontClientFactory` supports multiple client bindings. If more than one client configuration is specified, you can create a `WavefrontMultiClient` to send data to multiple Wavefront services.
 
 ## Option 1: Sending Data via the Wavefront Proxy
 
->**Prerequisite**
->Before your application can use a `WavefrontProxyClient`, you must [set up and start a Wavefront proxy](https://docs.wavefront.com/proxies_installing.html).
+### Prerequisites
 
-When sending data via the Wavefront proxy, you need to create a `WavefrontProxyClient`. Include the following information.
+Before you initialize the `WavefrontClient`, you must [set up and start a Wavefront proxy](https://docs.wavefront.com/proxies_installing.html).
```

Comment:
If you want to send data via the Wavefront proxy, you must set up.... before you initialize the WavefrontClient. 

Want to mention Proxies permission here?? Mentioning it with Direct Ingestion.